docs/api-reference/log/stats.md

```diff
@@ -1,65 +1,63 @@
 # Stats
 
-A class for tracking performance over time, in terms of frames per second or operations per second, suitable for integration with stats widgets.
-
-* "counters" for discrete events
-* "timers" for counting processing of a specific category
-
+A collection of statistic for tracking time or magnitude metrics.
 
 ## Usage
 
-Just inform your `Stats` instance when something happens. Then query it and get the FPS averages.
-```js
-const stats = new Stats();
-for (let i = 0; i < 100; i++) {
-  stats.bump('fps')
-}
-```
-
-Get the counters from a stats instance. Don't forget to reset it!
+Just create Stat objects (see `Stat` documentation) for various metrics.
 ```js
-const counters stats.getCounters();
-stats.reset();
+const stats = new Stats({id: 'my-stats'});
+const memoryUsage = stats.create('Mem');
+const executionTime = stats.create('Time');
+memoryUsage.addCount(1024);
+executionTime.timeStart();
+executionTime.timeEnd();
 ```
 
-
 ## Methods
 
 ### constructor
 
 `new Stats({id})`
 
-* `id` (`String`) - the id of the counter.
+* `id` (`String`) - the id of the `Stats` object.
+
 
+### create
 
-### addCounter
+Create a new stat tracker.
```

Comment:
What happens when I call `create` with a duplicate name? Can this be exchanged with `get`?

Reply:
`create` would replace the existing stat with a new one. `get`  only returns existing stats. 

Now that I think about it, I don't really like that API, since I've found that it's convenient to keep references to created stats. I'm thinking I'll update `get` to function as "get or create" (like the old `addCounter`) and remove the `create` method.